packages/http-service/src/request.js

```diff
@@ -0,0 +1,25 @@
+import Rx from "rxjs";
```

Comment:
`request` could be implemented by reshaping `stream` with some `rxjs` operators. That'd be DRY'er. I think it is better not to do this, but feel free to challenge that.

Reply:
:100: